ui/src/Components/ProfilePage.jsx

```diff
@@ -33,18 +37,67 @@ export default class ProfilePage extends React.Component {
       loading: true,
       profile: null,
       props: props,
+      admin:false,
+      verified:0,
     }
+
+    this.handlePendingVerParam = this.handlePendingVerParam.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
   }
 
   async componentDidMount()
   {
-    const url = `https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/viewProfile?id=${this.state.props.match.params.id}`;
+    const url = URL + `id=${this.state.props.match.params.id}`;
     const resp = await fetch(url);
     const data = await resp.json();
-    this.setState({profile: data, loading: false});
+    var adminState = querySearch(this.props.location.search).admin;
+    this.setState({profile: data, loading: false, admin: adminState});
+  }
+
+  handlePendingVerParam(event){
+    if(event.target.name === "true"){
+      this.setState({verified: 1});
+    } else{
+      this.setState({verified: 0});
+    }
   }
 
+  handleSubmit(event) {
+    event.preventDefault();
+    const url = URL + `id=${this.state.props.match.params.id}&verified=${this.state.verified}`;
+    fetch(url)
+    .then(res => res.json())
+    .then(res => {
+      this.setState({profile: res})
+    })
+    this.render()
+};
+
   render(){
+    const editOrValidateCTAs = ()=>{
+      if(this.state.admin == 1){ //validate CTAs displayed
+          // Currently displays "Took Pic File Uploaded" above the field, which seems incorrect. Until clarification, commenting out.
+          return(
+            <div>
+              <Form onSubmit={ this.handleSubmit }>
+                <div className="row-buffer side-buffer">
+                  <p className="right-align">Applicant Status:</p>
+                  <FormGroup className="right-align" check row>
+                    <Button type="submit" name="false" onClick={this.handlePendingVerParam} style={{margin: '5px'}}>More Information Required</Button>
+                    <Button type="submit" name="true" onClick={this.handlePendingVerParam} style={{backgroundColor: '#4CAF50'}}>Verify Candidate</Button>
+                  </FormGroup>
+                </div>
+              </Form>
+            </div>
+          ) 
+      } else{ //edit profile CTA
+        return (
+          <div className="right-align">
+            <Link to= {`/profiles/edit/${this.state.profile.result.profile_id}`}className="btn btn-secondary">Edit this profile</Link>
```

Comment:
minor: the button is stuck to the right side of the web page, can we format this similarly to other buttons (add a right margin b/w the button and the edge of the page)!

Reply:
<img width="937" alt="image" src="https://user-images.githubusercontent.com/37027135/100953876-4b845780-34c8-11eb-96fc-2a20546ef306.png">

Reply:
Fixed it